inloop/settings.py

```diff
@@ -206,3 +206,8 @@
 
 CONSTANCE_CONFIG.update(gitload_constance.config)
 CONSTANCE_CONFIG_FIELDSETS.update(gitload_constance.fieldsets)
+
+# Define the output path of the JPlag tests and their similarity factor
+JPLAG_JAR_PATH = str(BASE_DIR / "lib/jplag/jar/jplag.jar")
```

Comment:
The jplag.jar should be moved one level up.

Reply:
Two levels -> `str(BASE_DIR / "lib/jplag-x.y.z.jar")`, and the filename should contain the release number of JPlag (should be on their GH releases page).

Reply:
The release page says `lib/jplag-2.11.9-SNAPSHOT-jar-with-dependencies.jar`, but I think we can cut the name down to `lib/jplag-2.11.9-SNAPSHOT.jar`.